templates/commands/goldentest/verify.go

```diff
@@ -75,7 +77,21 @@ func (c *VerifyCommand) Run(ctx context.Context, args []string) (rErr error) {
 		return fmt.Errorf("failed to parse flags: %w", err)
 	}
 
-	testCases, err := parseTestCases(ctx, c.flags.Location, c.flags.TestNames)
+	templateLocations, err := crawlTemplateLocations(c.flags.Location)
+	if err != nil {
+		return fmt.Errorf("failed to parse template locations")
```

Comment:
Two issues here:
 - `parse` -> `crawl`
 - The call to fmt.Errorf should include `err` with `%w`, so the actual error is shown to the user